plugins/DuskVerb/tools/tuner/vintage_minisweep2.py

```
@@ -0,0 +1,93 @@
+        r = subprocess.run(
+            [sys.executable, str(REPO / "plugins/DuskVerb/tools/tuner/full_check.py"),
+             str(out_dir), str(ANCHOR), "--name", "BH", "--category", "Halls"],
+            capture_output=True, text=True, timeout=60)
+    except subprocess.TimeoutExpired:
+        return 999
+    for line in r.stdout.splitlines():
+        if "GATE(S) FAILED" in line:
+            try:
+                return int(line.split("✗")[1].strip().split()[0])
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)